presto-tests/src/test/java/com/facebook/presto/execution/resourceGroups/db/TestQueuesDb.java

```diff
@@ -250,9 +253,9 @@ public void testSelectorPriority()
         QueryId secondQuery = createQuery(queryRunner, dashboardSession(), LONG_LASTING_QUERY);
         waitForQueryState(queryRunner, secondQuery, FAILED);
 
-        resourceGroup = queryManager.getFullQueryInfo(secondQuery).getResourceGroupId();
-        assertTrue(resourceGroup.isPresent());
-        assertEquals(resourceGroup.get(), createResourceGroupId("global", "user-user", "reject-all-queries"));
+        DispatcherQueryManager dispatcherQueryManager = queryRunner.getCoordinator().getDispatcherQueryManager();
+        BasicQueryInfo basicQueryInfo = dispatcherQueryManager.getQueryInfo(secondQuery);
+        assertEquals(basicQueryInfo.getErrorCode(), QUERY_QUEUE_FULL.toErrorCode());
```

Comment:
Hmm this feels like a different test. Is there a reason why we won't also check the resource group?